README.txt

```diff
@@ -1,162 +1,89 @@
+INTRODUCTION
+------------
 
-= Vote Up/Down =
+The Vote Up/Down module allows the user to cast votes on some entities in
+a Drupal site with different widgets. This module uses Voting API to implement
+votes and Chaos Tools to provide AJAX support and make custom widgets.
+ * For a full description of the module, visit the project page:
+   https://drupal.org/project/vote_up_down
 
-== Overview ==
+ * To submit bug reports and feature suggestions, or to track changes:
+   https://drupal.org/project/issues/vote_up_down
 
-This module will let you make votes on some entities inside drupal with
-different widgets, by using Voting API and its some submodules: vud_node,
-vud_comment and vud_term.
 
-AJAX functionality, provided by ctools, is used for voting if the browser
-supports it.
+REQUIREMENTS
+------------
 
-=== Features ===
+This module requires the following modules:
 
-* Vote on the entities: nodes, comments and terms.
+ * Voting API (https://drupal.org/project/votingapi)
+ * Chaos Tools (https://drupal.org/project/ctools)
 
-To submit bug reports and feature suggestions, or to track changes:
-  http://drupal.org/project/issues/vote_up_down
 
-== Requirements ==
+FEATURES
+--------
 
-* Drupal modules:
-** voting_api: http://drupal.org/project/voting_api
-** ctools: http://drupal.org/project/ctools
+ * Voting on entities: nodes, comments and terms.
```

Comment:
Let's re-phrase it to be more general, implemented as a field, you may vote over any field-able entity, node, comment and term are examples of entities.